File: engine/engine-model/src/main/scala/tools/packet/field/effect/ForcedEquip.scala
package tools.packet.field.effect

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ForcedEquip(private var _team: Int) extends PacketInput {
  def team: Int = _team

  override def opcode(): SendOpcode = SendOpcode.FORCED_MAP_EQUIP
}

File: engine/engine-model/src/main/java/tools/packet/field/effect/ForcedEquip.java
package tools.packet.field.effect;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ForcedEquip(int team) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.FORCED_MAP_EQUIP;
   }
}